packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -147,7 +166,13 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result
         isReady(result);
         break;
       case MethodNames.START_CONNECTION:
-        startConnection((int) call.argument("handle"), result);
+        final int handle = (int) call.argument(MethodArgs.HANDLE);
+        boolean enableAlternativeBillingOnly = false;
+        if (call.hasArgument(MethodArgs.ENABLE_ALTERNATIVE_BILLING_ONLY)
```

Comment:
I would expect to see changes to https://github.com/flutter/packages/blob/d4c445422f2cd3f0627f575d85b59b559e0e9f69/packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_wrapper.dart#L119-L123 for this to be able to do anything. Am I looking at the wrong part of the Dart code? If not, this will never be true.

Reply:
You are correct, this slipped by me between last friday and yesterday. I would have caught it when adding functionality to the example app but am still working through the bureaucracy of shipping something I can test with end to end.